### ISSUE TITLE
it's not complete the info about "write a new driver" for netshot 

### ISSUE THREAD

**Issue by sjose165** [netfishers-onl/Netshot#228]
can u help with the complete info about write a new driver for netshot in the version 0.18.2. The section is edited and just have the head to the title, but no the instructions

**Comment by SCadilhac** [comment]
Indeed, this is yet to write :-) Contributions are welcome of course!

**Comment by PatrickDenis** [comment]
I would love to help, I need to make a driver for Aruba.  once I will completed it , i will try to make a document.
If anyone wants to join for the effort PM.
